gc/base/standard/ConfigurationGenerational.cpp

```diff
@@ -76,6 +76,26 @@ MM_ConfigurationGenerational::tearDown(MM_EnvironmentBase* env)
 	MM_ConfigurationStandard::tearDown(env);
 }
 
+bool
+MM_ConfigurationGenerational::initialize(MM_EnvironmentBase* env)
+{
+	MM_GCExtensionsBase* extensions = env->getExtensions();
+
+	#define DEFAULT_LOCAL_GC_TIME_RATIO_FOR_NURSERY_EXPAND		0.05
+	#define DEFAULT_LOCAL_GC_TIME_RATIO_FOR_NURSERY_CONTRACT	0.01
```

Comment:
In this case we could probably live without these #define's since we are storing the value in already nicely named GCExt fields. 
You can however comment declaration of dnssExpectedTimeRatioMaximum/Minimum and say those are GC time ratio thresholds for exapansion/contraction.

Reply:
With this change, `MM_GCExtensionsBase::dnssExpectedTimeRatioMaximum` is now a a `MM_UserSpecifiedParameterDouble`, initialized to 0.0 by default - with this, 0.01 and 0.05 values are no longer stored anywhere - Am I perhaps missing another field which contains this value?

Moving these #defines to `MM_GCExtensionsBase` may be a cleaner approach if you prefer that approach.

Reply:
I meant you could just do
- directly assign `dnssExpectedTimeRatioMaximum._valueSpecified = 0.05`
- comment at declaration point of dnssExpectedTimeRatioMaximum it's 'gc time ratio for expand'

you do it already for global heap resizing thresholds

`heapExpansionGCTimeThreshold._valueSpecified = 13;`

Or if you do want to keep the macros, then do it for the global ones too (and possibly move them to some *.h file that already consts (like omrgcconsts.h or similar). Bottom line, be consistent.

Does this help?

Reply:
Yes this clarifies what you meant! 
I will take the approach of simply initializing `dnssExpectedTimeRatioMaximum._valueSpecified = 0.05` before the if statement a few lines below, and add a comment.

I will also refactor `heapExpansionGCTimeThreshold._valueSpecified` initialization so it is consistent with the aformentioned value